submissions/lisnyak-o/popup-kottans/css/style.css

```diff
@@ -0,0 +1,162 @@
+body {
+  font-family: "Fira Sans", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  background-color: #fdfdfd;
+  color: rgb(138, 138, 138);
+  text-transform: uppercase;
+}
+.container {
+  max-width: 1230px;
+  margin: 0 auto;
+  padding: 0 15px;
+}
+.menu__wrapper {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+.menu {
+  margin-top: 30px;
+}
+.menu li + li {
+  padding-left: 10px;
+}
+.menu__item {
+  width: 60px;
+  height: 40px;
+}
+.menu__list a {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  width: 60px;
+  height: 40px;
+  color: #757575;
+  transition: all 0.5s ease;
+}
+.menu__list a:hover {
+  border-radius: 5px;
+  background-color: rgb(122, 138, 153);
+  text-align: center;
+  color: #fdfdfd;
+}
+.menu__list a:focus {
+  border: solid 1px rgb(122, 138, 153);
+}
+.menu__item-img {
+  width: 20px;
+  height: 20px;
+}
+.menu__item-photo {
+  border: 2px solid lightslategray;
+  width: 35px;
+  height: 35px;
+  border-radius: 50%;
+  object-fit: cover;
+}
+.menu__item-photo:hover {
+  border: 2px solid #fdfdfd;
+}
+input.popup-btn,
+div.content {
+  display: none;
+}
+input.popup-btn:checked + div.content {
+  display: block;
+}
+.popup {
+  position: relative;
+  z-index: 1;
+}
+.popup-base {
+  position: absolute;
+  top: 20px;
+  right: 8px;
+  width: 300px;
+  height: 500px;
+  background-color: #fdfdfd;
+  border: 1px solid #d3d3d3;
+  box-shadow: 5px 5px 10px 0px grey;
+}
+.square {
+  position: absolute;
+  top: -10px;
+  right: 117px;
+  height: 18px;
+  width: 18px;
+  border-color: #d3d3d3;
+  border-style: solid;
+  border-width: 1px 0px 0px 1px;
+  transform: rotate(45deg);
+  background-color: white;
+}
+.close-btn {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  z-index: -1;
+  opacity: 0.3;
+  background-color: #d3d3d3;
+  cursor: pointer;
+}
+.popup-base__scroll {
+  height: 100%;
+  overflow-x: hidden;
+  overflow-y: auto;
+}
+.scroll__items,
+.more__items {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  text-align: center;
+  padding-top: 20px;
+  padding-left: 27px;
+  height: calc(100% - 24px);
+}
+.more__items {
+  max-height: 50%;
+}
+.popup-base__scroll span {
+  display: block;
+}
+.scroll__item,
+.more__item {
+  width: 80px;
+  height: 80px;
```

Comment:
nothing wrong with these 80px, actually. I only suggest you to wrap the `<a>` elements into list so that it will be block elements. See the screenshot below, please. 
<img width="801" alt="Screenshot 2020-10-19 at 23 06 40" src="https://user-images.githubusercontent.com/25491366/96506936-1f3ab180-1261-11eb-8296-1aa9a02b5d38.png">

Reply:
So, I changed.